l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -158,6 +158,8 @@ def _setFormatoTrasmissione(self, cr, uid, company, context=None):
     def _setCodiceDestinatario(self, cr, uid, partner, context=None):
         if context is None:
             context = {}
+        #TODO check iPa code is required only for P.A. company
+        # but xml invoice can be make also for private company
```

Comment:
@archetipo fatturaPA is for PA. See http://www.fatturapa.gov.it/export/fatturazione/sdi/fatturapa/v1.1/Formato_FatturaPA_tabellare_V1.1.pdf : ```CodiceDestinatario``` is required (```1.1```) and contains
> codice dell'ufficio dell’amministrazione dello stato destinatario della fattura, definito dall'amministrazione di appartenenza come
riportato nella rubrica “Indice PA”.